model/build.gradle.kts

```diff
@@ -63,7 +63,9 @@ smallryeOpenApi {
   infoLicenseUrl.set("http://www.apache.org/licenses/LICENSE-2.0.html")
   schemaFilename.set("META-INF/openapi/openapi")
```

Comment:
just wondering:
why is this not `META-INF/openapi.yaml` or `src/main/resources/META-INF/openapi.yaml` or the like?

Reply:
There's already a generated openapi.yaml w/o the samples - that would conflict